src/components/KonnectorInstall.jsx

```diff
@@ -92,6 +92,7 @@ export const KonnectorInstall = ({
         {hasErrorExceptLogin && getErrorDescription({ t, error, connector })}
         {!!accountsCount &&
           !error &&
+          !isTimeout &&
```

Comment:
I am pretty sure that `isTimeout` is never set anywhere anymore, so that its value is always `undefined`. Did you succeed trying with a "real" timeout.

Furthermore, the logic now is not a timeout, but to know if the connection has been enqueued.

Reply:
I was testing that and that's true :/, some props need to be cleaned so 👍